integration_tests/tests/test_generate_source_some_tables.sql

```diff
@@ -0,0 +1,28 @@
+{% set raw_schema = generate_schema_name('raw_data') %}
+
+-- test all args
+{% set actual_source_yaml = codegen.generate_source(
+    schema_name=raw_schema,
+    database_name=target.database,
+    table_names=['data__a_relation'],
+    generate_columns=True,
+    include_descriptions=True
+) %}
+
+
+{% set expected_source_yaml %}
+version: 2
+
+sources:
+  - name: {{ raw_schema | trim }}
+    tables:
+      - name: data__a_relation
+        columns:
```

Comment:
#66 added the missing descriptions for the schemas and tables, so we need to add them to the expected output here:
```suggestion
  - name: {{ raw_schema | trim }}
    description: ""
    tables:
      - name: data__a_relation
        description: ""
        columns:
```